### PR TITLE
Bug/3

### PR DESCRIPTION
**Pull request by AlexSnowLeo**

### CODE REVIEW

**Review by Rewalon — COMMENTED** [review]
Reviewed, OK

**Comment by Rewalon** [comment]
But a Error is not fixed.

File: edf/FindForm.cs [hunk 1]
```diff
@@ -504,14 +504,34 @@ private string[] GetCheckFiles(ListView list)
         /// <returns><see langword="true "/>Для успешного запуска.</returns>
         private bool OpenFileByPath(string fileName)
         {
-            if (!File.Exists(listView.SelectedItems[0].SubItems[0].Text))
```
Comment (AlexSnowLeo) [inline edf/FindForm.cs:507]: Error been here

Comment (Rewalon) [reply edf/FindForm.cs:507]: now ok.

**Pull request merged**